reactComponents/src/types.ts

```diff
@@ -13,3 +15,244 @@ export {
   AccordionItemButton,
   AccordionItemPanel,
 };
+
+type CustomWrapperType = {
+  backgroundColor: string | undefined,
```

Comment:
https://www.typescriptlang.org/docs/handbook/advanced-types.html
![image](https://user-images.githubusercontent.com/43188883/166330722-61f1e523-914a-4736-a1e8-4f6b7228efeb.png)

This would be cleaner.